### DISCUSSION TITLE
How get all groups of className and its styles

### DISCUSSION THREAD

**Discussion by UNDERCOVERj** [tailwindlabs/tailwindcss#11361] [Help]
I want to get all groups of className and its styles in my project ，and list in my website，I want to search specific styles to get className

**Comment by babakfp** [comment]
Are looking for this? https://github.com/tailwindlabs/tailwindcss/discussions/11374

**Reply by UNDERCOVERj** [reply]
I want to display all class and styles in my local dev site
